Dockerfile

```
@@ -0,0 +1,22 @@
+FROM openjdk:8
+
+ARG APP_VERSION
+ENV APP_NAME=hbase-table-provisioner
+ENV APP_JAR=$APP_NAME-$APP_VERSION.jar
+ENV APP_HOME=/opt/$APP_NAME
+ENV USER=htp
+ENV GROUP=$USER
+
+RUN mkdir $APP_HOME
+WORKDIR $APP_HOME
+
+#COPY build/libs/*.jar ./$APP_NAME.jar
+
+RUN addgroup $GROUP
+RUN useradd -g $GROUP $USER
+
+#RUN chown -R $USER.$USER . && chmod +x ./$APP_NAME.jar
+
+USER $USER
+
+ENTRYPOINT ["sh", "-c", "sleep 600"]
```

not sure if a `sleep` will create a log entry, where as something like this will. That or an `echo` or something to help verify
```suggestion
ENTRYPOINT ["sh", "-c", "whoami"]
```